src/NzbDrone.Core/Datastore/Migration/165_remove_custom_formats_from_quality_model.cs

```diff
@@ -0,0 +1,141 @@
+using System;
+using System.Collections.Generic;
+using System.Data;
+using Dapper;
+using FluentMigrator;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Datastore.Converters;
+using NzbDrone.Core.Datastore.Migration.Framework;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(165)]
+    public class remove_custom_formats_from_quality_model : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Alter.Table("Blacklist").AddColumn("IndexerFlags").AsInt32().WithDefaultValue(0);
+            Alter.Table("MovieFiles").AddColumn("IndexerFlags").AsInt32().WithDefaultValue(0);
+
+            // Remove Custom Formats from QualityModel
+            SqlMapper.AddTypeHandler(new EmbeddedDocumentConverter<QualityModel164>());
+            Execute.WithConnection((conn, tran) => RemoveCustomFormatFromQuality(conn, tran, "Blacklist"));
+            Execute.WithConnection((conn, tran) => RemoveCustomFormatFromQuality(conn, tran, "History"));
+            Execute.WithConnection((conn, tran) => RemoveCustomFormatFromQuality(conn, tran, "MovieFiles"));
```

Comment:
Do we need to migrate it out of the Pending Releases table, parsedMovieInfo object as well? Could also fix #4041 at same time. If you look at the data in that issue, custom formats are present in the quality model there.